### PR TITLE
Currency: Added build scripts and updated build_info.json files for llvmlite

### PR DESCRIPTION
**Pull request by npanpaliya**
## Checklist

- [ ] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [ ] Have you validated script on UBI 9 container
- [ ] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: l/llvmlite/build_info.json [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-  "build_script": "llvmlite_ubi_8_7.sh",
-  "docker_build": false,
-  "validate_build_script": true,
-  "use_non_root_user": false,
-  "v0.41.1, v0.41.0": {
-    "build_script": "llvmlite_ubi_8_7.sh"
+   "maintainer": "npanpaliya",
+   "package_name": "llvmlite",
+   "github_url": "https://github.com/numba/llvmlite",
+   "version": "v0.41.1",
```
Comment (prachikurade1) [inline l/llvmlite/build_info.json:5]: Can add support for latest version - v0.43.0

File: l/llvmlite/llvmlite_ubi_9_3.sh [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Build_success_but_test_Fails"
+    exit 2
+else
+    echo "------------------$PACKAGE_NAME:build_&_test_both_success-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub  | Pass |  Both_Build_and_Test_Success"
+    exit 0
+fi
+
+deactivate
```
Comment (prachikurade1) [inline l/llvmlite/llvmlite_ubi_9_3.sh:124]: `deactivate` should be before exit 0.

**Review by prachikurade1 — CHANGES_REQUESTED** [review]

File: l/llvmlite/build_info.json (hunk 1 above)
Comment (prachikurade1) [reply l/llvmlite/build_info.json:5]: For currency the latest version is added and if we need a specific version (that does not work with the same script) then we add another script with the required version.
Sample PR - https://github.com/ppc64le/build-scripts/pull/4493

File: l/llvmlite/build_info.json [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
-    "build_script": "llvmlite_ubi_8_7.sh"
-  }
-}
\ No newline at end of file
+   "maintainer": "npanpaliya",
+   "package_name": "llvmlite",
+   "github_url": "https://github.com/numba/llvmlite",
+   "version": "*",
+   "default_branch": "main",
+   "build_script": "llvmlite_v0.41.1_ubi_9_3.sh",
```
Comment (shubham-dayma-ibm) [inline l/llvmlite/build_info.json:7]: Good to set latest version's build script here(v0.43.0)

**Review by shubham-dayma-ibm — CHANGES_REQUESTED** [review]

File: l/llvmlite/build_info.json [hunk 4]
```diff
@@ -1,15 +1,23 @@
...
+   "package_dir": "l/llvmlite",
+   "docker_build": true,
```
Comment (nikhil-kalbande) [inline l/llvmlite/build_info.json:9]: if we are not adding dockerfile.. lets not make docker_build flag true.

File: l/llvmlite/build_info.json [hunk 5]
```diff
@@ -1,15 +1,23 @@
...
+   "validate_build_script": true,
+   "use_non_root_user": false,
+   "*": {
```
Comment (nikhil-kalbande) [inline l/llvmlite/build_info.json:12]: If no docker build support you can remove this whole block  
   "*": {
     "dir": "linux",
      "base_docker_image": "base",
      "base_docker_variant": "Not defined"
   },

**Review by nikhil-kalbande — CHANGES_REQUESTED** [review]

**Review by ghatwala — APPROVED** [review]
lgtm

**Pull request merged**